src/main/scala/mesosphere/marathon/MarathonSchedulerActor.scala

```diff
@@ -580,6 +581,7 @@ private[this] object SchedulerActions {
       case (Some(av), Some(bv)) => fn(av, bv)
       case _ => default
     }
+    // TODO ju replaceable with ```a.taskStatus, ...``` ?
```

Comment:
Yes, definitely. We want to sort by state, then timestamp. The timestamp _should_ be the timestamp of the state, but the stagedAt will do as well as long as the compared states are equal.